### PR TITLE
Add v26 migrate docs

### PR DESCRIPTION
**Pull request by DracoLi**

### CODE REVIEW

File: migrate/v0_26/migrate.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Kava 16 Upgrade Instructions
+
+## Software Version and Key Dates
+
+- The version of `kava` for Kava 16 is `v0.26.0`
+- The Kava 16 chain will be shutdown with a `SoftwareUpgradeProposal` that
+  activates at approximately 15:00 UTC on April 27th, 2024.
```
Comment (nddeluca) [inline migrate/v0_26/migrate.md:7]: Is this a typo? Do we mean April 24th?  Or is this a placeholder?

File: migrate/v0_26/migrate.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Prior to applying the Kava 16 upgrade, validators are encouraged to take a full data snapshot at the upgrade height before proceeding. Snap-shotting depends heavily on infrastructure, but generally this can be done by backing up the .kava directory.
+
+It is critically important to back-up the .kava/data/priv_validator_state.json file after stopping your kava process. This file is updated every block as your validator participates in consensus rounds. It is a critical file needed to prevent double-signing, in case the upgrade fails and the previous chain needs to be restarted.
+
+In the event that the upgrade does not succeed, validators and operators must downgrade back to v0.25.x of the Kava software and restore to their latest snapshot before restarting their nodes.
+
+### Coordination
+
+If the Kava 16 chain does not launch by April 28th, 2024 at 00:00 UTC, the launch should be considered a failure. In the event of launch failure, coordination will occur in the [Kava discord](https://discord.com/invite/kQzh3Uv).
```
Comment (nddeluca) [inline migrate/v0_26/migrate.md:77]: Ditto on date

**Review by nddeluca — CHANGES_REQUESTED** [review]
Looks good but think the dates are wrong or we should write them as TBD

File: migrate/v0_26/migrate.md (hunk 1 above)
Comment (DracoLi) [reply migrate/v0_26/migrate.md:7]: Yeah placeholder

Comment (DracoLi) [reply migrate/v0_26/migrate.md:7]: will change to TBD

**Pull request merged**